graviti/portex/field.py

```diff
@@ -85,6 +97,147 @@ def rename(self, old_name: str, new_name: str) -> None:
         self._keys.__setitem__(self._keys.index(old_name), new_name)
 
 
+class ConnectedFields(MutableMapping[str, PortexType]):
+    """Fields composed of ImmutableFields and MutableFields.
+
+    Raises:
+        ValueError: When there as repeated field names.
+
+    Arguments:
+        fields: The ImmutableFields and MutableFields.
+
+    """
+
+    def __init__(self, fields: Iterable[Union[ImmutableFields, MutableFields]]) -> None:
```

Comment:
do we need a base class of ImmutableFields and MutableFields

Reply:
I don't think it's necessary currently.
And the `ImmutableFields` could be the base class.